docs/html-jsx.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML to JSX</title>
+  <!-- Github Pages lacks basic functionality such as permanent server-side
+       redirects, so the best we can do is a meta redirect -->
+  <meta http-equiv="refresh" content="0; url=http://magic.reactjs.net/htmltojsx.htm" />
```

Comment:
Ahh, I didn't even notice there was a `redirect` layout.

I like `.htm`... It's one less character, and consistent with pretty much everything else having a three-letter file extension (well, apart from `.js`). The extra `l` doesn't provide any extra value  :smile:  haha